federated-etl.md

```diff
@@ -8,7 +8,18 @@ Below is the configuration guide using **Kubecost ETL Federation**
 
 Federated ETL gives teams the benefit of federating multiple Kubecost installations into one view without dependency on Thanos.
 
-For environments that already have a Prometheus instance, this ETL Federation may be preferred because the only dependency will be a single Kubecost pod for monitored cluster.
+There are two primary advantages for using ETL Federation:
+1. For environments that already have a Prometheus instance, Kubecost only requires a single pod per monitored cluster
+2. Many solutions that aggregate Prometheus metrics (like Thanos) often to not scale well in large environments
```

Comment:
Perhaps adjust this?

> Many solutions that aggregate Prometheus metrics (like Thanos) often to not scale well in large environments

to

> Many solutions that aggregate Prometheus metrics (like Thanos), are often expensive to scale in large environments

Reply:
much better. will commit. 

Any other changes based on the work you are doing? I'll wait for your approval before committing. 

Not urgent.